xapian-core/tests/testdata/apitest_cluster.txt

```diff
@@ -3,3 +3,13 @@ A computer cluster consists of a set of loosely or tightly connected computers t
 The components of a cluster are usually connected to each other through fast local area networks ("LAN"), with each node (computer used as a server) running its own instance of an operating cluster system. In most circumstances, all of the nodes use the same hardware and the same operating system, cluster although in some setups (i.e. using Open Source Cluster Application Resources (OSCAR)), different operating systems can be used on each computer, clusterand/or different hardware.
 
 They are usually deployed to improve performance and availability over that of a single computer, while typically being much more cost-effective than single computers of comparable speed or availability.
+
```

Comment:
I had no clue about the license requirements. I'd rather write up a few lines which wouldn't require any license's and will help me write test data according to the clusterers being tested. Thanks for pointing this out.